.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,5 +20,6 @@ This PR closes #{issue_number}.
 * [ ] I ran `task all`, and it passed.
 * [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
 * [ ] I checked comments rendering with `task godocs`.
-* [ ] (for maintainers only) I set Reviewers, Assignee, Labels, Project and project's Sprint fields.
+* [ ] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)),
+      Assignee, Labels, Project and project's Sprint fields.
 * [ ] I marked all done items in this checklist.
```

Comment:
You didn't do it for the current PR! :D

Reply:
Sorry, WDYM?

Reply:
Already fixed. When I was checking the PR, this item hadn't been checked for it.